API/common/utils.py

```diff
@@ -19,8 +19,7 @@
 import subprocess
 import time
 import pyrebase
-import console
-import lumpy
+from API.common import console, lumpy
```

Comment:
Could you please separate the system and the custom module imports (by a newline)?